### ISSUE TITLE
Cannot Fly?

### ISSUE THREAD

**Issue by gabessdsp** [gravitylow/AntiCheat#13]
I gave myself 

anticheat.exempt
anticheat.fly

but I still can't fly.

I also used x-ray and no x-ray stats came up for me? This is before and after I added anticheat.exempt.

But please fix the flying problem.

Here is my permissions file just in case it is my fault(Note:I am under the Emperor Group):
http://pastie.org/3987955

**Comment by gravitylow** [comment]
Are you using creative flymode or a mod? By "can't fly" do you mean it teleports you down and sends warnings to the console?

**Comment by gabessdsp** [comment]
I am using MyCraft modmanager flying. Which is the same as zombe's flymod. If you see in the permissions file I also added anticheat.zombe.fly (or whatever it is). But yes it teleports me down and send messages, increasing my warning level.

**Comment by gravitylow** [comment]
Does it work if you have op?
It sounds more like a permissions issue to me :3 What permissions plugin are you using?

**Comment by gabessdsp** [comment]
I am using PEX and that's why I supplied the permissions.yml file. But if I make myself op then yes I can fly. But I need it to work without op.

**Comment by gravitylow** [comment]
I understand, I'm just trying to figure out the issue, but it sounds more like a plugin issue rather than an actual bug inside anticheat. (I recently got rid of PEX and switched to bPermissions and saved myself from a lot of anger, errors, and bugs that don't seem to get cleaned up in PEX)

**Comment by gabessdsp** [comment]
I figured out what was wrong and I solved every problem I was having except for this one. My inheritance was setup wrong. But now I have trying this on multiple accounts, now with just the /fly command which is creative flying. This type of flying is being blocked even though my user even has anticheat.exempt

**Comment by gabessdsp** [comment]
AHAH! I have more info, so with those permission nodes added I CAN fly but ONLY when I am in CREATIVE mode. I need to be able to fly, it is still blocking me with all types of flying when I am in survival mode. I need to be able to fly in survival mode.

**Comment by gabessdsp** [comment]
I feel like this is an anticheat-side problem now as well because I also have the anticheat.admin pNode, when I use commands they still work but I also get a "You don't have permissions to do this", although the command still works.

**Comment by gravitylow** [comment]
Sounds like maybe it's an older build? Try this one: http://ci.h31ix.net/job/AntiCheat/129/ if you aren't already. Since the official 1.1 release we have removed all creative checks and based it purely on fly speed. Also, AntiCheat will only ever give you a failed permissions check message of "Insufficient Permissions", so if the message you described is actually word for word the message, it's not AntiCheat.

**Comment by gabessdsp** [comment]
Okay I changed to that build.....but now I still can't fly and it just tells me I have INsufficient permissions to do anything even though I have anticheat.admin

**Comment by SuperSpyTX** [comment]
I would reset your permissions configuration and hit basics with the PEX wiki.  And reviewing it carefully.

**Comment by gabessdsp** [comment]
I deleted my PEX and installed a completely new one and that's how I fixed my other perm problems I was having. But here is what I have been doing for the past 30 minutes.

I download BRAND NEW JARS and brand new craftbukkit.
I started a different server on port 25566 so I could run it side-by-side.
I installed PEX and then ANTICHEAT
I started the server once so that the config's generated.
I used the generated permissions file and added anticheat.exempt
I then logged on to the server, in survival mode it still wouldn't let me fly, but in creative mode it let me fly.

Therefore there seems to be a problem with anticheat.
